tailormap-components/projects/core/src/lib/feature-form/state/form.reducer.ts

```diff
@@ -15,7 +15,21 @@ const onSetFeature = (state: FormState, payload : ReturnType<typeof FormActions.
   feature: payload.feature,
 });
 
-const onSaveFeature = (state: FormState, payload:  ReturnType<typeof FormActions.setSaveFeatures>) : FormState => ({
+const onSetNewFeature = (state: FormState, payload: ReturnType<typeof FormActions.setNewFeature>): FormState => ({
+  ...state,
+  features: [
+    {
+      ...state.features[0],
```

Comment:
Is this correct? Are you *always* updating the first one? Then its ok, but still it could be better to lookup the right feature first based on id or something